File: demo/src/main/java/com/example/android/entity/Conversation.java
package com.example.android.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("conversations")
public class Conversation {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long initiatorId;
    private Long receiverId;
    private Long productId;
    private Long requestId;
    private String type;
    private String status;
    private String lastMessage;
    private Date lastMessageTime;
    private Date createdAt;
    private Date updatedAt;
}

File: app/src/main/java/com/example/xiaoyi/model/Conversation.kt
package com.example.xiaoyi.model

data class Conversation(
    val id: Long = 0L,//会话ID
    val initiatorId: Long = 0L,//发起者用户ID
    val receiverId: Long = 0L,//接收者用户ID
    val productId: Long? = null,//关联的商品ID
    val status: String = "",//会话状态
    val createdAt: String = "",//创建时间
    val lastMessage: String = "",//最新消息内容
    val lastMessageTime: String = "",//最新消息时间
    val otherUserId: Long = 0L,//对方用户ID
    val otherUserName: String = "",//对方用户名
    val otherUserAvatar: String = "",//对方用户头像
    val unreadCount: Int = 0//未读消息数量
)